### DISCUSSION TITLE
I am getting error when doing an API request - write EPROTO 751624:error:1000012e:SSL routines:OPENSSL_internal:KEY_USAGE_BIT_INCORRECT:..\..\third_party\boringssl\src\ssl\ssl_cert.cc:607:

### DISCUSSION THREAD

**Discussion by MukeshDevan** [thunderclient/thunder-client-support#1266] [General]
I am getting error when doing an API request - write EPROTO 751624:error:1000012e:SSL routines:OPENSSL_internal:KEY_USAGE_BIT_INCORRECT:..\..\third_party\boringssl\src\ssl\ssl_cert.cc:607:

**Comment by rangav** [comment]
Hi @MukeshDevan 
can you share the following
- extension version
- vscode version
- Are you using any custom SSL certificates

**Comment by MukeshDevan** [comment]
extension version: v2.10.2
vscode version: 1.81.1
Are you using any custom SSL certificates: yes, custom self signed certificates

**Reply by rangav** [reply]
Thanks for sharing.

- is it working before in previous versions?
- Did you see any errors in logs (Output window TC filter)

I am not expert in the ssl cert, may be google it or post in stackoverflow

**Reply by Arunjoshua1410** [reply]
may be a port error
